apps/i18n/common/en_us.json

```diff
@@ -1316,7 +1317,7 @@
   "syncClever": "Sync students from Clever",
   "syncGoogleClassroom": "Sync students from Google Classroom",
   "syncingYourStudents": "Syncing Your Students",
-  "syncingYourStudentsDescription": "If your Code.org section is ever out of date with your list of students in {loginType}, click on the \"Sync students from {loginType}\" button on the \"Manage students\" tab: ",
+  "syncingYourStudentsDescription": "If your Code.org section is ever out of date with your list of students in {loginType}, click on the \"Sync students from {loginType}\" button on the <a href={url}>\"Manage students\"</a> tab: ",
```

Comment:
this pattern of allowing HTML inside translated text is scary and I think we should look into alternatives. I know that others in this situation have not liked the options, so there may not be a great answer. I think there might be a way to do it using `dangerouslySetInnerHTML`. @Hamms do you know what the best practice is here?

Reply:
There's no great way to do this just yet, but the best practice for right now is to do this with markdown rather than html and use the UnsafeRenderedMarkdown component.

The long-term plan is to use the redaction/restoration process for markdown to defend against link injection via translations, and for the UnsafeRenderedMarkdown component to be updated to leverage remark to defend against script injection.

Reply:
thanks! i am using the `UnsafeRenderedMarkdown` component, so i think we're good there. i tried to insert the hyperlink using markdown syntax, but it didn't work, so i used HTML instead. i was probably doing it wrong -- what's the best way to insert a markdown hyperlink into a string?

this is what i tried:
`"If your Code.org section is ever out of date with your list of students in {loginType}, click on the \"Sync students from {loginType}\" button on the [\"Manage students\"]({url}) tab: "`

Reply:
I think this is how to do it using dangerouslySetInnerHTML: https://github.com/code-dot-org/code-dot-org/blob/1cad5ac667f9ed1a3aa31063e6d06bd21a78e2bd/apps/src/code-studio/components/ReportAbuseForm.jsx#L125-L132

The corresponding string definition would look like this: 
https://github.com/code-dot-org/code-dot-org/blob/40c5ac0bef2cda7ae546f552feacdef00abbe27c/dashboard/config/locales/en.yml#L885

Reply:
That looks correct to me! What happened when you tried that?

Reply:
@Hamms the string displayed as a string literal, like this:

`"If your Code.org section is ever out of date with your list of students in {loginType}, click on the \"Sync students from {loginType}\" button on the [\"Manage students\"](/my/url/here) tab: "`

Reply:
ah, it's the quotes. if i just have `[Manage Students]({url})` instead of `[\"Manage Students\"]({url})`, it works fine. the quotes aren't really necessary anymore, so i'm going to just delete them. thank you both for your help!

Reply:
Much better 👍 thanks for fixing!